src/__tests__/unit/controllers/router-store/test.tsx

```diff
@@ -38,11 +38,13 @@ const mockHistory = {
 
 const mockRoutes = [
   {
+    basePath: '',
```

Comment:
This should not be here anymore, right?

Reply:
Yup. Removed now.

Reply:
Created a new dir for integration tests and added two new tests. `__tests__/integration/test.tsx`